src/interactions/rbe.jl

```diff
@@ -0,0 +1,153 @@
+struct RBEInteraction{T} <: ExTinyMD.AbstractInteraction
+    α::T
+    cutoff::T
+    ε::T
+end
+
+Base.show(io::IO, interaction::RBEInteraction) = print(io, "RBEInteraction with α = $(interaction.α), cutoff = $(interaction.cutoff), ε = $(interaction.ε)")
+
+RBEInteraction(;α::T = 1.0, cutoff::T = 3.5, ε::T = 1.0) where T = RBEInteraction(α, cutoff, ε)
+
+function acceptance_probability(m_star::Float64, α::Float64, L::Float64)
+    if m_star == 0
+        return erf(1 / (2 * sqrt(α * L^2 / π^2)))
+    else
+        sqrt_val = sqrt(α * L^2 / π^2)
+        return 0.5 * (erf((abs(m_star) + 0.5) / sqrt_val) - erf((abs(m_star) - 0.5) / sqrt_val))
+    end
+end
+
+function mh_sample(α::Float64, L::Float64)
+    sample = 0.0
+    while sample == 0.0
+        x_star = rand(Normal(0, sqrt(α * L^2 / (2 * π^2))))
+        m_star = Float64(round(Int, x_star))
+        q = acceptance_probability(m_star, α, L)
+        if rand() < q && m_star !=0
+            sample =  m_star
+            break
+        end
+    end
+    return sample
+end
+
+function sampling(α, L, n)
```

Comment:
creating a vector of $k$ and the prob is especially expensive, store them as part of the RBEInteraction